--- scripts/package/builddeb	2024-06-24 15:46:22.781494357 +1000
+++ scripts/package/builddeb.patched	2024-06-24 15:46:56.369820382 +1000
@@ -48,7 +48,7 @@
 
 	# Create the package
 	dpkg-gencontrol -p$pname -P"$pdir"
-	dpkg-deb $dpkg_deb_opts ${KDEB_COMPRESS:+-Z$KDEB_COMPRESS} --build "$pdir" ..
+	dpkg-deb ${DPKG_FLAGS} $dpkg_deb_opts ${KDEB_COMPRESS:+-Z$KDEB_COMPRESS} --build "$pdir" ..
 }
 
 install_linux_image () {
